config_files/config_lorem_ipsum.yaml

```diff
@@ -5,7 +5,7 @@ modalities_setup:
 
 wandb:
   project_name: modalities
-  mode: ONLINE
+  mode: OFFLINE
```

Comment:
Shouldn't the default be `ONLINE`? Otherwise, the user has to manually call the syncing script.

Reply:
I am not sure what this setting does. However we keep on encountering issues with WandB if the tests (this config is only for the unittests) run on different devices, which is always based on the fact that not everybody uses WandB (especially the CI test setup). Therefore deactivating this in the tests makes much sense in my opinion. 
I deactivated them beforehand by using a different subscriber, but if this here is also an option, I do very much appreciate it.

Reply:
This setting stores the WandB logs only locally and does not sync them automatically. The user has to call a WandB syncing routine to push the logs to the WandB server. 

If this config is solely used for testing, then we can leave it as is. At the same time, we should have another config, which logs to WandB automatically to avoid confusion for the users.